src/framework/theme/components/menu/menu.component.ts

```diff
@@ -50,8 +51,20 @@ export class NbMenuItemComponent implements DoCheck, AfterViewInit, OnDestroy {
 
   private alive = true;
   toggleState: NbToggleStates;
+  isLtr$: Observable<boolean>;
+  isRtl$: Observable<boolean>;
 
-  constructor(private menuService: NbMenuService) {}
+  constructor(private menuService: NbMenuService,
+              private directionService: NbLayoutDirectionService) {
+    this.isLtr$ = this.directionService.onDirectionChange()
```

Comment:
It's better to move initialization code into lifecycle method, such as `ngOnInit`.